shared/js/background/components/resource-loader.js

```diff
@@ -234,7 +243,7 @@ export default class ResourceLoader extends EventTarget {
      */
     async modify (func) {
         await this.ready
-        const newValue = func(this.data)
+        const newValue = func(this.data) || this.data
```

Comment:
(Just so the `.modify()` calls in the Devtools component don't have to pointlessly return the reference to this.data back again.)

Reply:
Fair enough, Done.